local-interchaintest/configs/logs.json

```diff
@@ -1,33 +1,33 @@
 {
-  "start_time": 1717066959,
+  "start_time": 1727093314,
   "chains": [
     {
       "chain_id": "localcosmos-1",
       "chain_name": "localcosmos-1",
-      "rpc_address": "http://0.0.0.0:55225",
-      "rest_address": "http://0.0.0.0:55227",
-      "grpc_address": "0.0.0.0:55226",
-      "p2p_address": "0.0.0.0:55229",
+      "rpc_address": "http://0.0.0.0:56590",
+      "rest_address": "http://0.0.0.0:56592",
+      "grpc_address": "0.0.0.0:56591",
+      "p2p_address": "0.0.0.0:56594",
```

Comment:
Are those being automatically changes? or the ports are always the same?

Reply:
these are randomly assigned, but internally in local-interchain they are mapped to static ones